samples/unit-tests/axis/type-logarithmic/demo.js

```diff
@@ -264,4 +264,36 @@ QUnit.test('Y axis minimum got stuck (#3353)', function (assert) {
         postUpdatesTick,
         "Y minimum value should not be changed when updating yAxis type"
     );
+});
+
+QUnit.test('Negative values on log axes', function (assert) {
```

Comment:
Can we test all possible cases? 

- xAxis: log, yAxis: log:
  - x: positive, y: positive,
  - x: positive, y: negative,
  - x: negative, y: positive,
  - x: negative, y: negative

- xAxis: log, yAxis: linear:
  - x: positive, y: positive,
  - x: positive, y: negative,
  - ...
- ...